include/classes/Savant2/Savant2.php

```diff
@@ -994,50 +994,87 @@ function findFile($type, $file)
 	* @throws object An error object with a SAVANT2_ERROR_ASSIGN code.
 	* 
 	*/
-	
-	function assign()
+	public function assign()
 	{
-		// this method is overloaded.
-		$arg = func_get_args();
-		
-		// must have at least one argument. no error, just do nothing.
-		if (! isset($arg[0])) {
-			return;
-		}
+		// get the arguments; there may be 1 or 2.
+		$arg0 = @func_get_arg(0);
```

Comment:
Be more specific on the variable name to specify what is expected to be received on this argument, rather than using a generic unclear name like `arg0`. The same comment applies across: always use a meaningful variable name.

Reply:
The variable name was declared by the creator of the library ,not me